gapic/src/main/com/google/gapid/MainWindow.java

```diff
@@ -91,6 +92,7 @@
   protected final Map<MainTab.Type, Action> viewTabs = Maps.newHashMap();
   protected final Set<MainTab.Type> hiddenTabs = Sets.newHashSet();
   protected Action editCopy;
+  private StatusBar statusBar;
```

Comment:
> seems like it's rather restrictive.

Not really, but I'm OK with punting on doing the research on it. A cursory look suggests it uses a `ContributionManager` and `ContributionItem`s... Either way, it doesn't look like it does all that much stuff to render, so may not even be worth using.